circuits/circom/poseidon/poseidon.circom

```diff
@@ -0,0 +1,16 @@
+pragma circom 2.0.0;
+
+// https://github.com/weijiekoh/circomlib/blob/feat/poseidon-encryption/circuits/poseidon.circom
+include "./poseidon-cipher.circom";
+
+template Poseidon_OLD(nInputs) {
```

Comment:
it would conflict with circomlib's one if I didn't change the name (and yes technically is the old poseidon implementation which is what we have been using until now and won't change because we would need different zkeys)

Reply:
I see, thanks for the explanation @ctrlc03. Should we create an issue to upgrade the implementation we're using (i.e. to use circomlib's)?

Reply:
I think it's something we should be tracking on zk-kit also, so that the poseidon-cipher circuits can be upgraded to use the new version (I've tried to implement with circomlib's new poseidon algorithm, but was unsuccessful in my first try)